src/services/app_info.js

```diff
@@ -4,7 +4,7 @@ const build = require('./build');
 const packageJson = require('../../package');
 const {TRILIUM_DATA_DIR} = require('./data_dir');
 
-const APP_DB_VERSION = 158;
+const APP_DB_VERSION = 159;
```

Comment:
As mentioned above, DB migration is not necessary here.

Reply:
Understood, will update the same 👍